Xamarin.Essentials/Map/Map.mac.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Map
+    {
+        internal static Task PlatformOpenMapsAsync(double latitude, double longitude, MapLaunchOptions options)
```

Comment:
You can open mac's Maps app with a URL like "http://maps.apple.com/?ll=45.5200,-122.6819".